components/Map.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import FeatureCounts from '../components/FeatureCounts';
+import moment from 'moment';
```

Comment:
I'd actually recommend seeing if `date-fns` can do this for you: https://hackernoon.com/why-you-should-choose-date-fns-over-moment-js-in-your-nodejs-applications-116d1a709c43

`moment` has a notoriously large code footprint.